### PR TITLE
Random password generator 

### LINKED ISSUE #842: Random password generator GUI
# Aim
To create a random password 

# Details

A simple GUI based password generator.

### PR DESCRIPTION
**Pull request by tanvi355**
# Description
Random password generator GUI

## Fixes #(issue_no)
Fixes #842 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Random Password Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## RANDOM PASSWORD GENERATOR
```
Comment (kaustubhgupta) [inline Random Password Generator/README.md:1]:
```suggestion
# RANDOM PASSWORD GENERATOR
```

**Review by kaustubhgupta — APPROVED** [review]
Looks good! Just one minor change:

**Comment by tanvi355** [comment]
> Looks good! Just one minor change:

Done!

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚩

**Pull request merged**